src/components/TodoTotal.js

```diff
@@ -0,0 +1,15 @@
+import { filterTodos } from "../utils/helpers.js";
+
+export default class TodoTotal {
+  constructor(store, $app) {
+    this.store = store;
+    this.$app = $app;
+  }
+  render() {
+    const { view, todos } = this.store.getState();
+    //prettier-ignore
+    const curViewTodos = view === "all" ? todos 
```

Comment:
```suggestion
    const currentViewTodos = view === "all" ? todos 
```
reduce 메소드를 사용할때만 cur 사용하기